engine/src/witness/btc.rs

```diff
@@ -128,6 +128,8 @@ where
 		.chunk_by_vault(vaults.clone(), scope)
 		.deposit_addresses(scope, unfinalised_state_chain_stream, state_chain_client.clone())
 		.await
+		.private_deposit_channels(scope, state_chain_stream.clone(), state_chain_client.clone())
```

Comment:
I was going to follow your suggestion (though I didn't entirely agree), but as I was making changes I realised that this would be problematic: `btc_deposits` expects private channels to be present. We could probably think of a workaround, but seems not worth it considering that we are planning to add support for this very soon (probably even in the same release).

Reply:
yes true